shared-bindings/wifi/Radio.c

```diff
@@ -117,7 +117,7 @@ const mp_obj_property_t wifi_radio_hostname_obj = {
               MP_ROM_NONE},
 };
 
-//|     mac_address: ReadableBuffer
+//|     mac_address: str
```

Comment:
`ReadableBuffer` is correct here. It's not a human-readable string, it's the bytes of the MAC address.
```suggestion
//|     mac_address: ReadableBuffer
```